packages/junto-utils/react/PerspectiveContext.tsx

```diff
@@ -36,7 +38,9 @@ const initialState: ContextProps = {
     members: {},
     channels: {},
   },
-  methods: {},
+  methods: {
+    getMessageProfile: (did: string) => null
```

Comment:
I would maybe rename this to just getProfile, as the PerspectiveContext could be used to get profile information in other perspective-views that do not necessarily care about chat messages.